content/en/logs/log_collection/csharp.md

```diff
@@ -307,10 +307,13 @@ var log = new LoggerConfiguration()
     .CreateLogger();
 ```
 
-You can override the default behavior by manually specifying the following properties in the parameters: `endpoint`, `port`, `useSSL`. You can also specify the `source`, `service`, `host`, and custom tags:
+**Note**: To send logs to Datadog EU site, set the `url` property to `https://http-intake.logs.datadoghq.eu`
+
+You can also override the default behaviour and forward logs in TCP by manually specifying the following required properties: `url`, `port`, `useSSL`, and `useTCP`. Optionally, specify the `source`, `service`, `host`, and custom tags.
```

Comment:
Should these options link somewhere to define what they do and their format? This sentence sounds a bit odd as is as well.